tests/test_configurer.py

```diff
@@ -74,4 +74,15 @@ def test_dsconf_pass_keys():
 
     conf_dict['spam'] = 'eggs'
     with pytest.raises(KeyError):
-        conf = DataServerConf.from_conf_dict(conf_dict, required_keys, allowed_keys)
\ No newline at end of file
+        conf = DataServerConf.from_conf_dict(conf_dict, required_keys, allowed_keys)
+
+def test_dsconf_legacy_plugin_keys():
+    conf_dict = {
+                 'data_server_url': 'bacon',
+                 'data_server_port': 'eggs',
+                 'data_server_remote_cache': 'spam',
+                 'dispatcher_mnt_point': 'spam',
+                 'dummy_cache': 'spam!',
+                }
+
+    conf = DataServerConf.from_conf_dict(conf_dict)
```

Comment:
What is the addition compared to test_dsconf_integral?

Reply:
Ah I put the test quickly at the end, which is unfortunate, since usually I would show the problem with a failing test and then make a fix. Here is I started the right way: https://github.com/oda-hub/dispatcher-app/pull/31 - it is just for the test. Then we can deal with the fix of #30  here.